README.md

```diff
@@ -97,8 +97,79 @@ Kusto language plugin for the Monaco Editor. It provides the following features
     };
 ```
 
+#### ESM (webpack example)
```

Comment:
This is a very long list of things to do...
Is there any way to remove steps here by changing the code?

Reply:
Probably some work can be done to make steps 3-4 be smaller,
though this work changes the packing of the package and should be checked carefully against requireJs load. My guideline to this PR was to enable new ESM functionality and to stick with the current packing of AMD modules.